eds/styles/styles.css

```diff
@@ -307,6 +310,10 @@ main {
 
   /* sections */
 
+  .section > .default-content-wrapper {
+    text-align: center;
+  }
```

Comment:
We should just allow centering in the Doc. I don't see that coming across in a quick test. I worry about setting it as default behavior -- we'll just end up with lots of local overrides.